x-pack/plugins/fleet/server/routes/app/index.ts

```diff
@@ -133,7 +134,7 @@ export const generateServiceTokenHandler: RequestHandler<
       token?: GenerateServiceTokenResponse;
     }>({
       method: 'POST',
-      path: `_security/service/elastic/${serviceAccount}/credential/token/token-${Date.now()}`,
+      path: `_security/service/elastic/${serviceAccount}/credential/token/token-${uuidv4()}`,
```

Comment:
thanks for the contribution. @swapanmandi it seems elasticsearch is already generating a uuid if the name is not passed, so I think we can use that behavior and directly call `.../credential/token` without a name